Model/Config/GridXmlToArrayConverter.php

```diff
@@ -452,11 +457,12 @@ private function convertMassActionsConfig(\DOMElement $root)
     private function convertMassActionConfig(\DOMElement $actionElement): array
     {
         return filter(merge(
+            $this->getAttributeConfig($actionElement, 'id'),
```

Comment:
I've added this to provide it in css-class styling and in general I think it would be a good idea (for e.g. having javascript functions before execution) on mass-elements.
What do you think?

Reply:
Good approach - sounds perfect